hera_cal/abscal.py

```diff
@@ -285,6 +321,11 @@ def delay_lincal(self, medfilt=True, kernel=(1, 11), time_ax=0, freq_ax=1, verbo
 
         time_avg : boolean, if True, average resultant antenna delays across time 
 
+        window : str, window to enact on data before FFT for dly solver, options=['blackmanharris', 'hann', None]
+            None is a top-hat window.
+
+        edge_cut : int, number of channels to flag at each band edge in FFT window
```

Comment:
I think it's slightly clearer if the default is 0 rather than None, since that makes the type consistent.